wrappers/C#_High_Level_Wrapper/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
The license should be the top-level license, and by contributing, you agree to use the existing public domain license

Reply:
Actually I can delete the license,